### PR TITLE
Adding list command

### PR DESCRIPTION
**Pull request by breezetall**
User can now refresh the list panel by using the listtask command. The command for listing tasks is:
 ```
listtask
```

## Things to note
The listtask command will not display anything if the list is empty.
 
This PR is linked to #4.

### CODE REVIEW

**Review by nobodyishappy — APPROVED** [review]
Looks good to me!

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -163,6 +166,21 @@ public TaskList getTaskList() {
         return taskList;
     }
 
+    /**
+     * Returns an unmodifiable view of the list of {@code Person} backed by the internal list of
+     * {@code versionedAddressBook}
+     */
```
Comment (Yskie) [inline src/main/java/seedu/address/model/ModelManager.java:172]: Should this comment should be regarding tasks instead of person?

**Review by Yskie — APPROVED** [review]
Other than the issues with the JavaDoc, rest looks good!

File: src/test/java/seedu/address/logic/commands/ListTaskCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    /* Test for find method */
+    /*
+    @Test
+    public void execute_listIsFiltered_showsEverything() {
+        showPersonAtIndex(model, INDEX_FIRST);
+        assertCommandSuccess(new ListCommand(), model, ListCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+    */
```
Comment (chin-herng) [inline src/test/java/seedu/address/logic/commands/ListTaskCommandTest.java:40]: ditto on removing unused code.

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (breezetall) [reply src/main/java/seedu/address/model/ModelManager.java]: Fixed! Thank you.

**Review by Kaya3842 — APPROVED** [review]
No issues found. LGTM!

**Pull request merged**